roles/ci_gen_kustomize_values/templates/nova04delta/edpm-nodeset-values/values.yaml.j2

```diff
@@ -0,0 +1,100 @@
+---
+# source:  nova04delta/edpm-nodeset-values/values.yaml.j2
+{% set _ipv = cifmw_ci_gen_kustomize_values_ip_version_var_mapping %}
+{% set instances_names = []                                                            %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _original_services = _original_nodeset['services'] | default([])                %}
+{% for _inst in cifmw_networking_env_definition.instances.keys()                       %}
+{%     if _inst.startswith('compute')                                                  %}
+{%         set _ = instances_names.append(_inst)                                       %}
+{%     endif                                                                           %}
+{% endfor                                                                              %}
+{% set inst_stop_idx = (instances_names | length) // 2                                 %}
+{% set nodeset_one_instances = instances_names[:inst_stop_idx]                         %}
+data:
+  baremetalSetTemplate:
+    provisioningInterface: null
+  ssh_keys:
+    authorized: {{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | b64encode }}
+    private: {{ cifmw_ci_gen_kustomize_values_ssh_private_key | b64encode }}
+    public: {{ cifmw_ci_gen_kustomize_values_ssh_public_key | b64encode }}
+  nova:
+    migration:
+      ssh_keys:
+        private: {{ cifmw_ci_gen_kustomize_values_migration_priv_key | b64encode }}
+        public: {{ cifmw_ci_gen_kustomize_values_migration_pub_key | b64encode }}
+  nodeset:
+    ansible:
+      ansibleUser: "zuul"
+      ansibleVars:
+        edpm_fips_mode: "{{ 'enabled' if cifmw_fips_enabled|default(false)|bool else 'check' }}"
+        timesync_ntp_servers:
+          - hostname: "{{ cifmw_ci_gen_kustomize_values_ntp_srv | default('pool.ntp.org') }}"
+        edpm_bootstrap_command: |
+          # root CA
+          pushd /etc/pki/ca-trust/source/anchors/
+          curl -LOk {{ cifmw_install_ca_url }}
+          update-ca-trust
+          popd
+
+          # install rhos-release repos
+          dnf --nogpgcheck install -y {{ cifmw_repo_setup_rhos_release_rpm }}
```

Comment:
TBH `--nogpgcheck` is not fine, but I know that in other places is done similar way.